ajax/libs/epub.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "epub.js",
+  "filename": "epub.min.js",
+  "description": "Render ePub documents in the browser, across many devices",
+  "keywords": [
+    "epub",
+    "ebooks"
+  ],
+  "author": "fchasen@gmail.com",
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/futurepress/epub.js.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "+(epub|hooks|reader)*",                                                                                                                                                            
+          "!(libs)/**/*" 
```

Comment:
We still can add `epub_no_underscore.js*` in v0.1.7~v0.2.12
Let's use the pattern `!(libs){,**/*}` to exclude the files which we don't need and include the files which can be added.

The whole auto-update config is
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/futurepress/epub.js.git",
    "fileMap": [
      {
        "basePath": "build",
        "files": [
          "!(libs){,**/*}"
        ]
      }
    ]
  },
```

Many thanks.



ref: https://github.com/futurepress/epub.js/issues/273#issuecomment-138759591